src/rust/engine/process_execution/Cargo.toml

```diff
@@ -5,6 +5,14 @@ name = "process_execution"
 authors = ["Pants Build <pantsbuild@gmail.com>"]
 publish = false
 
+[[bin]]
+name = "sandboxer"
+path = "src/sandboxer_server_main.rs"
```

Comment:
Maybe put these into their own subdirectories with a `main.rs`? The `main.rs` pattern is generally what the main file for a Rust binary is called. I.e., `src/sandboxer/server/main.rs` and `src/sandboxer/client/main.rs`. It also provides some nice isolation from the rest of the `process_execution` crate.

Or beter yet, maybe just make the sandboxer be its own crate?

Reply:
Sandboxer can't easily be its own crate because it uses `ManagedChild` from `process_execution` , and is in turn used by other code in `process_execution`. I looked into factoring `ManagedChild` out of its crate, but figured it wasn't worth the effort. Can still do it though.

As for the separate subdirs for the binaries, I could do that.